.circleci/config.yml

```diff
@@ -1,584 +1,29 @@
-    executor: node
-    steps:
-      - attach_workspace:
-          at: .
-      - run:
-          name: Auth With NPM
-          command: echo "//registry.npmjs.org/:_authToken=$npm_TOKEN" > .npmrc
-      - run:
-          name: Publish unreleased package versions to npm
-          command: npx lerna publish from-package --ignore-scripts --yes
-      - slack_notify_fail
-      - slack_notify_success
+# the path-filtering orb is required to continue a pipeline based on
+# the path of an updated fileset
+orbs:
+  path-filtering: circleci/path-filtering@0.0.2
 
 workflows:
-  version: 2
-
-  build_and_test:
-    jobs:
-      - install:
-          context: web-core
-          filters:
-            branches:
-              ignore: [ 'gh-pages' ]
-      - danger-checks:
-          name: danger-checks
-          context: web-core
-          filters:
-            branches:
-              ignore: ['gh-pages']
-          requires:
-            - install
-      - build:
-          name: build
-          context: web-core
-          filters:
-            branches:
-              ignore: ['gh-pages']
-          requires:
-            - danger-checks
-      - bundlewatch:
-          name: bundlewatch
-          context: web-core
-          filters:
-            branches:
-              ignore: ['gh-pages']
-          requires:
-            - build
-      - unit-tests:
-          matrix:
-            parameters:
-              component_type: ['tools', 'services', 'atoms', 'molecules', 'organisms', 'pages', 'templates']
-          context: web-core
-          filters:
-            branches:
-              ignore: ['gh-pages']
-          requires:
-            - build
-      - browser-tests:
-          matrix:
-            parameters:
-              component_type: ['atoms', 'molecules', 'organisms', 'pages']
-          context: web-core
-          filters:
-            branches:
-              ignore: ['gh-pages']
-          requires:
-            - unit-tests-<< matrix.component_type >>
-      - publish-to-npm:
-          context: web-core
-          filters:
-            branches:
-              only: master
-          requires:
-            - browser-tests-atoms
-            - browser-tests-molecules
-            - browser-tests-organisms
-            - browser-tests-pages
-
-  deploy-storybook:
+  # the always-run workflow is always triggered, regardless of the pipeline parameters.
+  always-run:
     jobs:
-      - deploy-storybook:
-          context: web-core
-          filters:
-            branches:
-              only: master
+      # the path-filtering/filter job determines which pipeline
+      # parameters to update.
+      - path-filtering/filter:
+          name: check-updated-files
+          # 3-column, whitespace-delimited mapping. One mapping per
+          # line:
+          # <regex path-to-test> <parameter-to-set> <value-of-pipeline-parameter>
+          mapping: |
+            ^(?=.*^package\.json|\.circleci).*$ global-change true
```

Comment:
set `global-change` to true if root package.json / .circleci folder has changed.